offchain/jd/provider/ctf_provider.go

```diff
@@ -0,0 +1,173 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/avast/retry-go/v4"
+	csav1 "github.com/smartcontractkit/chainlink-protos/job-distributor/v1/csa"
+	ctf_jd "github.com/smartcontractkit/chainlink-testing-framework/framework/components/jd"
+	"github.com/smartcontractkit/chainlink-testing-framework/framework/components/postgres"
+
+	"github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink-deployments-framework/offchain"
+	"github.com/smartcontractkit/chainlink-deployments-framework/offchain/jd"
+)
+
+// CTFOffchainProviderConfig holds the configuration to initialize the CTFOffchainProvider.
+type CTFOffchainProviderConfig struct {
```

Comment:
```suggestion
type CTFProviderConfig struct {
```

This is inline with the onchain provider naming